client/src/modules/user/AuthBox/components/SignInForm.jsx

```diff
@@ -61,11 +83,18 @@ class SignInForm extends PureComponent {
     );
   };
 
+  onPolicyChange = () =>
+    this.setState(({ isPolicyAccepted }) => ({
+      isPolicyAccepted: !isPolicyAccepted
+    }));
+
   emailValidator = value =>
     validateWith(value => isEmail(value))('user.auth.input.email.invalid')(
       value
     );
 
+  handleLogin = () => this.setState({ pending: true });
```

Comment:
this method sets pending to true, it is not responsible for handling login,
rename or remove and use `this.setState` where necessary